src/axom/multimat/multimat.cpp

```diff
@@ -788,6 +983,12 @@ void MultiMat::convertFieldToSparse(int field_idx)
     return;
   }
 
+  if(!m_fieldBackingVec[field_idx]->isOwned())
+  {
+    throw std::runtime_error("Multimat error: cannot convert unowned field \"" +
+                             m_fieldNameVec[field_idx] + "\" to sparse layout.");
```

Comment:
Is this (and the `throw` at (new) line 1030) the reason for the try/catch at (new) lines 897--905 and 880--889?

Reply:
A different strategy would be to use `SLIC_WARNING` or `SLIC_ERROR` for invalid operations.

Reply:
Switched over to SLIC for error handling.